src/components/common/TreeTag.js

```diff
@@ -56,6 +57,11 @@ function TreeTagComponent({ TreeTagValue, title, icon }) {
             variant="body2"
           >
             {TreeTagValue}
+            {title === 'Age'
+              ? ' Years'
+              : title === 'Diameter at Breast Height'
+              ? ' cm'
+              : ''}
```

Comment:
@pierrelstan do you think we can put the logic outside so the tree page will handle the conditions and put raw text into this component? I feel that would relieve the knowledge burden for this component.

Reply:
Yes, it's possible. I will make the changes.